shun2218-dev/portfolio-site
# Reflect Dependabot Updates from Main Branch to Develop Branch

This pull request is intended to reflect the dependency module vulnerability fixes made by Dependabot from the main branch to the develop branch.

### Changes
- Updated dependencies including the latest security fixes
- Enhanced security

### Background
The dependency vulnerability fixes by Dependabot were applied in the main branch. To maintain the development flow, these updates need to be reflected in the develop branch as well. This will enhance security in the development environment.

### Verification Steps
1. After merging this pull request, verify that the dependencies are correctly updated.
2. Ensure all tests pass successfully.
3. Update related documentation or configuration files as needed.

Thank you for your attention and review.
